src/main/java/net/snowflake/client/core/arrow/BitToBooleanConverter.java

```diff
@@ -14,6 +14,13 @@
 public class BitToBooleanConverter extends AbstractArrowVectorConverter {
   private BitVector bitVector;
 
+  /**
+   * Constructor
```

Comment:
let's skip comment `Constructor` - parameters description is enough, here and in other places